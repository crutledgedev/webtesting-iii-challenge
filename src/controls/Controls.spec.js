// Test away!
import React from 'react';
import { render } from '@testing-library/react';
import Controls from './Controls';

test("Controls renders without crashing", () => {
    render(<Controls/>);
});

test('Toggle States', () => {
    const locked = false;
    const closed = true;
    const toggleLocked = true;
    const toggleClosed = true;
    const tree = render(<Controls
        locked = {locked}
        closed = {closed}
        toggleLocked = {toggleLocked}
        toggleClosed = {toggleClosed}
        />
        )
        expect(tree.baseElement).toMatchSnapshot()
})