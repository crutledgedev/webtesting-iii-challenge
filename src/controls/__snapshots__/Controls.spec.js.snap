// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toggle States 1`] = `
<body>
  <div>
    <div
      class="controls panel"
    >
      <button
        class="toggle-btn"
      >
        Lock Gate
      </button>
      <button
        class="toggle-btn"
      >
        Open Gate
      </button>
    </div>
  </div>
</body>
`;
